### ISSUE TITLE
OSX Big Sur app redesign

### ISSUE THREAD

**Issue by edwardbattistini** [iglance/iGlance#195]
Hi,

I've just downloaded your software. Really cool alternative to all the paid versions. As I did not pay for your software, I made a few options for an OSX icon style. Let me know what you think. 

[OSX icon.zip](https://github.com/iglance/iGlance/files/4819978/OSX.icon.zip)

If you want some support at some point for those little UI bits feel free to get in touch

**Comment by edwardbattistini** [comment]
@D0miH @Moneypulation I know its a little bit odd to get contributions like that but what do you think? about the icons? Not interested? let me know

**Comment by D0miH** [comment]
Hey @edwardbattistini,
sry for the late reply, I was very busy. At first glance those icons look pretty good. 
I will take a closer look at it over the weekend. 
Since I'm not really good at graphics programs like Affinity, your help is definitely very welcome 👍🏼😊

**Comment by Moneypulation** [comment]
> @D0miH @Moneypulation I know its a little bit odd to get contributions like that [...]

Absolutely not! We are happy to receive any contribution and welcome people to participate in the design and development of this app, not just the code. Like @D0miH, I also didn't have much time to look into it yet.

From a pure visual look, you icons look really good and definitely better than iGlance's current icon. Though I have to say, that I'm missing a recognition factor in your icons. I kind of like the string "iG" in the current icon, because you'll recognize the app icon pretty fast due to it (in my opinion). But I'm also open for other ideas. Perhaps, you could recreate the current icon making it look more professional and maybe add some details (?)

**Comment by D0miH** [comment]
So after looking at the icons I have to say they look very nice when applied to the app!
<img width="178" alt="Screenshot 2020-06-26 at 10 00 45" src="https://user-images.githubusercontent.com/36657497/85834601-f7947a80-b793-11ea-946f-52db9334d91d.png">

My favourite one is the first one where there is no separation between the glare and the cogwheel because it looks very minimalistic. Since it is not very common on mac apps to have letters in the app icon and instead just have a minimalistic visual pleasing icon I really like your designs. Therefore, when changing the icon of the app I don't think we should stick to the "iG" letters in the logo.

Why did you choose a cogwheel? I am asking this because a cogwheel creates the impression on my side that you can "change" some settings on your mac using this app.

**Comment by edwardbattistini** [comment]
oh! i did not get a notification for you answers! I'm glad @D0miH likes it! @Moneypulation I see your point around the iG there is something about the iG that is recognizable, and you know I could play around with it a bit more when I have a sec. 

The reasoning behind the Eye and the Cog was very literal: iGlance allows me to SEE the stats about how my machine works THE COG and because its kind of a utility I thought the the combination would work well.

We can def iterate IMO it was just a little token of appreciation for your open source project. 
I originally worked in affinity because it was the file format of the project but I think moving to figma will help a lot to make faster iterations. I'll see what I can do in the next few days

Thanks for your replies!

**Comment by edwardbattistini** [comment]
Also, we probably need to explore the new Big Sur style for the icons

**Comment by D0miH** [comment]
> Also, we probably need to explore the new Big Sur style for the icons

Yeah, definitely! Everything is rounder in Big Sur. 

> The reasoning behind the Eye and the Cog was very literal: iGlance allows me to SEE the stats about how my machine works THE COG and because its kind of a utility I thought the the combination would work well.

I see. I like the idea👍🏼

> We can def iterate IMO it was just a little token of appreciation for your open source project.
I originally worked in affinity because it was the file format of the project but I think moving to figma will help a lot to make faster iterations. I'll see what I can do in the next few days

Thank you very much! This is why I open sourced the project such that everyone can enjoy the app for free as well as everyone can give input and help to further develop the app. Just use your favorite tool as long as I can open the files somehow later on. Since I saw that figma has a free tier this is fine :)

**Comment by edwardbattistini** [comment]
Here is a Figma link with the icons. I'm planning to have the whole UI added in too. So I can suggest maybe some icons and UI tweaks. 

https://www.figma.com/file/mY3HMmzx8mc61EGFTbHdza/iGlance?node-id=0%3A1

**Comment by D0miH** [comment]
Wow the two new icons look so awesome! I really like the idea with the little inscription on the cogwheel.👍🏼😁
Really looking forward for your UI design changes :)

**Comment by edwardbattistini** [comment]
Had a play with the UI I've redone all the tab icons ready to go if you like them!

![Preview](https://user-images.githubusercontent.com/6288298/86948707-7222aa00-c145-11ea-86f3-7ee506350f61.jpg)

**Comment by D0miH** [comment]
WOW🤩 THIS LOOKS AWSOME!
Fits really well to the overall Big Sur look. You even redesigned the menu bar icons. Looks really great!

I think with the new design we could even remove the name of the app from the main window. This would allow us to have more space for the content. What do you think?

I know that at the moment there is way to much free space on the "Dashboard". My idea was that with future updates the menu bar icons can be arranged using drag and drop like in [iStat Menus](https://bjango.com/mac/istatmenus/). The main idea is that in the detailed views (e.g. "Processor", "Memory" etc.) the user can choose which menu bar icons he wants. Then on the dashboard view he can arrange the order in which the "Processor", "Memory" info should be displayed. iStat Menus called this the "Combined" view:
<img width="1002" alt="Screenshot 2020-07-08 at 19 14 28" src="https://user-images.githubusercontent.com/36657497/86949647-41477280-c14f-11ea-9dae-518cd23b1953.png">
So basically my idea was that this free space is used to display these bars there for dragging and dropping. From a design perspective, do you think we should display the overall system info (like uptime, cpu etc.) together with this drag and drop bars in one view? Or should we instead (like in iStats Menus) add another entry to the sidebar where the user can arrange the items?

**Comment by Moneypulation** [comment]
Wow, both the logo and the redesigned menu bar icons look really awesome. Thanks for your time and effort! Regarding @D0miH's earlier point about the "iG" string, he has point that it's not usual for macOS apps to have a logo like that. In addition, I really like your new version of the cogwheel logo. It looks really sharp and has a nice touch with the inscription

**Comment by edwardbattistini** [comment]
I use to love istat Menu, I am big fan of the [founder!](https://twitter.com/marcedwards) He's kind of pixel nerd 😄 
I wish I could pull the same UI he does! 

It's really up to you. I feel like iStat needs to justify the price. From my perspective I don't know enough about the target audience of those tools. I like those tools because they give me that that sense of control of my laptop, but at the end of the day if my MacBook is about to take off ( fan full power) I kind of know already...

I'm really wondering what kind of true game changer iGlance could bring in the monitoring app game...

About all your comments Lets try everything 😄

**Comment by edwardbattistini** [comment]
Are you guys considering rebuilding the UI with SwiftUI? I've been checking[ this guy on twitter,](https://twitter.com/jsngr) he's making magic with it and the layout is so easy to build. I'm really beginner in Swift so don't know what it would take but the Mocks the guy builds are madd

**Comment by D0miH** [comment]
Last time I looked at SwiftUI it was still pretty rudimentary. But I suppose this has changed now seeing all the cool designs [Jordan Singer](https://twitter.com/jsngr) is creating. I will definitely have a look at SwiftUI and see if it can be used to realize the drag and drop thin I want to integrate into the app.

**Comment by edwardbattistini** [comment]
Just in case you guys had not seen this. Also let me know if you need the icons in a specific format 


![Light-Preview-2](https://user-images.githubusercontent.com/6288298/87556732-e6fa6480-c6ae-11ea-9279-ad94fcdb304a.jpg)

![Light-Preview-1](https://user-images.githubusercontent.com/6288298/87556730-e5c93780-c6ae-11ea-9474-e9234ed45d3b.jpg)

![Dark-Preview-2](https://user-images.githubusercontent.com/6288298/87556726-e530a100-c6ae-11ea-91af-813dc360e6de.jpg)
![Dark-Preview-1](https://user-images.githubusercontent.com/6288298/87556713-e1048380-c6ae-11ea-8b6a-ec7bd26f66fe.jpg)

**Comment by D0miH** [comment]
Looks sick!
I guess the format of the sidebar icons can be pdf. This way we can preserve the vector data. 
The file format of the menu bar icons should be png. I guess the icons should have the same height as before (18 pixels high).
In what program did you create the icons? Also in Figma?

Regarding your previous question @edwardbattistini:
> Are you guys considering rebuilding the UI with SwiftUI?

I looked into it and it seems like SwiftUI needs macOS 10.15 or higher. Because I would like to support also older macOS version I think we won't be using SwiftUI

**Comment by edwardbattistini** [comment]
Here you go.

[iGlance - Sidebar icons.zip](https://github.com/iglance/iGlance/files/4926042/iGlance.-.Sidebar.icons.zip)
[iGlance - Menubar icons.zip](https://github.com/iglance/iGlance/files/4926041/iGlance.-.Menubar.icons.zip)

**Comment by D0miH** [comment]
Great! Thank you very much!
I will start to look into how to realise your design in code :)

**Comment by edwardbattistini** [comment]
let me know if I can help

**Comment by Anders-E** [comment]
Just got this update and just wanted to say that this is some great work @edwardbattistini ! Loving the new look.
